### PR TITLE
[WIFI-2850] Microservice processes in Docker containers should run as a dedicated user

### PR DESCRIPTION
**Pull request by oblom0v**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -47,9 +49,11 @@ RUN cmake --build . --config Release -j8
 
 FROM alpine
 
+RUN addgroup -S ucentralfms && adduser -S -G ucentralfms ucentralfms
+
 RUN mkdir /ucentral
 RUN mkdir /ucentralfms-data
-RUN apk add --update --no-cache librdkafka curl-dev 
+RUN apk add --update --no-cache librdkafka curl-dev mariadb-connector-c libpq unixodbc
```
Comment (dunaev-opsfleet) [inline Dockerfile:56]: IMO we should put this in different PR as it is clearly doesn't fit into scope of this PR

Comment (oblom0v) [reply Dockerfile]: agree, reverted the changes and will put them in a separate PR.

**Review by dunaev-opsfleet — APPROVED** [review]
